RocketBot2/RocketBot2.csproj

```diff
@@ -1082,7 +1082,7 @@ foreach (var item in filesToCleanup)
   </Target>
   <PropertyGroup>
     <PostBuildEvent>cd "$(TargetDir)"
-copy /y /b RocketBot2.exe "$(SolutionDir)\PoGo.Necrobot.Window\$(OutDir)\RocketBot2.exe" /b
+REM copy /y /b RocketBot2.exe "$(SolutionDir)\PoGo.Necrobot.Window\$(OutDir)\RocketBot2.exe" /b
```

Comment:
You need the opinion of more devs besides me. Pinging @jjskuld for Approval on doing this for GUI